### PR TITLE
🐛 Source Zoom: update cloud registry metadata

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What

A small fix to Source Zoom's metadata. This connector is not currently available in Cloud, and will remain so until we are able to [set up a valid test environment to ensure its stability](https://github.com/airbytehq/airbyte-internal-issues/issues/1824). The metadata does not reflect this fact, so our docs currently incorrectly list it as available in Cloud.

Zoom Cloud Status

## Note on Acceptance Tests

Because we do not have a working test environment, no PR for this connector can pass CAT tests at this time. I'm requesting an approve-and-merge on this PR since it's just updating an incorrect metadata field and not making any changes to the connector itself.

### CODE REVIEW

File: airbyte-integrations/connectors/source-zoom/metadata.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ data:
   name: Zoom
   registries:
     cloud:
-      enabled: true
+      enabled: false
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-zoom/metadata.yaml:13]: I believe we need to update our documentation, but we shouldn't release it without conducting thorough testing.

**Review by lazebnyi — CHANGES_REQUESTED** [review]

**Comment by DanyloGL** [comment]
It was decided to create new Zoom test account. Account was created and added to LastPass. For now it's free and could be used for testing only 2 out of 19 streams. Waiting for decision regarding what paid plan should be chosen.

**Comment by DanyloGL** [comment]
New Zoom test account credentials were added to GSM. All streams except those that require Webinars were seeded (8 out of 19).

**Comment by ChristoGrab** [comment]
Closing this as work on the connector has moved to #35369

**Pull request closed without merging**